### PR TITLE
Fix running in non-home directory

### PR DESCRIPTION
**Pull request by Anonymous941**
If you run this from any other folder, the X11 launcher won't appear

### CODE REVIEW

**Comment by zjoasan** [comment]
True, but the addon consists of two parts. The Launch-addon's code right now needs hard coded paths (/home/osmc/x11-start) to the "bash scripts part" i'm slowly working on an version 2 of this script, with an option to switch between Kodi and X, without killing Kodi completly and to integrate the bash script into addon file structure. But due to lack of time, I have had to put the project on pause. Alternatively do a variable where to install path can be an argument to the installer, before version uppgrade.

So in short, it will probbly fark up the addon. And give me some urgency to return to v.2 of this addon. But not before next week. Still want me to accppt teh PR?

**Comment by Anonymous941** [comment]
Well it's up to you...  The only reason I made this PR is because I didn't know that and thought others might make the same mistake, but maybe a note in the README would be better if this messes things up

**Comment by zjoasan** [comment]
Since it breaks the current setup, I will leave it hanging. But I will mention in the Readme that it expects to install under osmc's home folder, due to launcher add-on is currently hard coded to a folder in /home/osmc/x11-start. Might work in a "select install user" into the install script. But it's low on the priority list. Hope you understand.
